From 8ca396f8501f2fd96dae8892b0cc911fa0caae0d Mon Sep 17 00:00:00 2001
From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date: Sun, 11 Jul 2021 03:26:55 +0200
Subject: [PATCH 80/98] dt-bindings: arm: amlogic: Add the Tronsmart S82

Tronsmart S82 is a smart TV box with the following specifications:
- Amlogic S802 SoC
- 2GB DDR3
- 8GB or 16GB eMMC flash
- micro SD card slot
- built-in SDIO WiFi
- Bluetooth 4.0 on the same module as WiFi
- 10/100Mbit/s Ethernet

Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>

[dts: rebased onto 6.6.144 for 6.6-lts - tcu,fernsehfee3 isn't in the enum
 yet at this baseline (added by a later unrelated commit); inserted
 tronsmart,s82 right after minix,neo-x8 instead]
---
diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index e29f10080..b953f1f21 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -35,6 +35,7 @@ properties:
         items:
           - enum:
               - minix,neo-x8
+              - tronsmart,s82
           - const: amlogic,meson8
 
       - description: Boards with the Amlogic Meson8m2 SoC
